docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+  sorc/bufr_datacount.fd/datacount.f
+  sorc/bufr_listdumps.fd/CMakeLists.txt
+  sorc/bufr_listdumps.fd/inc_w3unpkb7.inc
+  sorc/bufr_listdumps.fd/iw3unpbf.f
+  sorc/bufr_listdumps.fd/listdumps.f
+  sorc/bufr_listdumps.fd/w3unpkb7.f
+  sorc/bufr_listdumps.fd/wordlength.f
+  sorc/bufr_prepmods.fd/CMakeLists.txt
+  sorc/bufr_prepmods.fd/bufr_prepmods.f
+  sorc/bufr_prepmods.fd/clonlat.f
+  sorc/bufr_prepmods.fd/mbuoyb.f
+  sorc/bufr_prepmods.fd/mnemonics.f
+  sorc/bufr_prepmods.fd/obname.f
+  sorc/bufr_prepmods.fd/raddate.f
+  sorc/bufr_prepmods.fd/ssmipn.f
+  sorc/bufr_prepmods.fd/tideg.f
+  sorc/bufr_remorest.fd/CMakeLists.txt
+  sorc/bufr_remorest.fd/remorest.f
+  sorc/prepobs_prepanow.fd/CMakeLists.txt
+  sorc/prepobs_prepanow.fd/prepanow.f
+  triggers/ (30)
+  ush/CMakeLists.txt
+A ush/build.sh
+  ush/bufr_avgdata.sh
+  ush/bufr_datacount.sh
+  ush/bufr_prepmods.sh
+  ush/bufr_remorest.sh
+  ush/dumpmb
+  versions/README
+  versions/build.ver
+M versions/run.ver
+M versions/VERSION
+------------------
+M-modified; A-added; D-deleted
+
+JOBS file changes
+-----------------
+ jobs/JOBSPROC_AIRNOW_DUMP
+ Correct COMIN path 
+
+ jobs/JOBSPROC_NAM_PREP
+ Correct COMINtcvital path; Turn off SYNDATA in prepbufr
+
+SCRIPTS file changes
+--------------------
+ scripts/exglobal_dump.sh
+ scripts/exrap_dump.sh
+ scripts/exurma_dump.sh
+ scripts/exrtma_dump.sh
+ scripts/excdas_dump.sh
+ Add dump groups: gmi1cr, snocvr, saldrn, subpfl
+ Add Sentinel-6 data to gpsro (via dev to prod tank update)
+ Add Metop-B/C AMVs data from tank b005/xx081 to satwnd dump group
+
+ scripts/exglobal_makeprepbufr.sh
+ Add subpfl and saldrn to nsst bufr dump
+
+ scripts/exnam_makeprepbufr.sh
+ Comment out COMSPtcvital path to turn off SYNDAT for nam
+
+USH script changes
+------------------
+ ush/build.sh
+ Move build.sh from obsolete /build-obsproc to /ush
+
+DOCS file changes
+-----------------
+ docs/README.md
+ Update build instructions  
+
+VERSIONS file changes
+----------------
+ versions/run.ver
+ Secure turn off SYNDAT for nam
+
+FIX code changes
+-------------------
+ No changes.
+
+Source code changes
+-------------------
+ No changes
+
+Output changes
+--------------
+ Generate new dumps:
+ *gmi1cr*, *snocvr*, *saldrn*, *subpfl*
+ Update dumps:
+ *gpsro*, *satwnd* and *nsstbufr (*nsstbufr is generatd only in global) 
+
+Compute Resource Information
+----------------------------
+ Memory usage was increased, from 60 to 120gb, when testing with triggers/jglobal_dump.wc2.pbs
+
+Disk Space Changes 
+------------------
+ The newly generated dumps ( *gmi1cr*, *snocvr*, *saldrn*, *subpfl* ) vary in size from 1Mb to 200Mb
+ depending on network, cycle time, time dump window. 0 size dumps are possible
+ The updated dumps (*gpsro*, *satwnd* and *nsstbufr) are up to ~25% larger in size
+ For examples, see: /lfs/h2/emc/stmp/iliana.genkova/CRON/RELv1.1.0/com
+ 
+Shared Software 
+---------------
+ obsproc needs modules bufr-dump and prepobs in order to run
```

Comment:
This is documentation, not Release Note material.

Reply:
it's in the /doc folder. further consideration to your comments will be given when time allows